### PR TITLE
Features/associations

### PR DESCRIPTION
**Pull request by hansderly**
In this PR, I:
✅ Created a Classroom class and its constructor.
✅ Created Setter and Getter for @Label.
✅ Created the has-many relationship between Classroom and Student.
✅ Created the belongs-to relationship between Classroom and Student.
✅ Created a Book class and its constructor.
✅ Created a Getter and Setter for the title and author.
✅ Created a Rental class and its constructor.
✅ Created a Getter and Setter for a date.
✅ Created the has-many side of Book and Rental.
✅ Created the belongs-to side of Rental and Book.
✅ Created the has-many side of Person and Rental.
✅ Created the belongs-to side of Rental and Person.
✅ Modified the constructor of Rental so Book and Person are set in it.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 require_relative './decorators'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```
Comment (mukhammadrizooff) [inline person.rb:5]: - [ ] [REQUIRED] Good job so far! Please change the place of `:rentals` to `attr_accessor` 🙏🏼 

```ruby
# *Getters and Setters for @rentals should be present
  attr_accessor :name, :age, :rentals
```

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @hansderly, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @hansderly,

Good job working on the changes highlighted by the previous reviewer 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:
✅ You used Gitflow correctly
✅ Linter checks passed 
✅ Set up associations correctly 👍🏽

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by hansderly** [comment]
Thank you @AmaduKamara.😊

**Pull request merged**